### PR TITLE
Improve the printing of CSS at-rules

### PR DESCRIPTION
**Pull request by evilebottnawi**
Also improve tests and refactor.

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@lydell @azz friendly ping, seems problem in 4 `production`

**Comment by lydell** [comment]
Do you have an example of something that prints better after this?

File: tests/css_atrule/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -1032,30 +1110,30 @@ salamander
 }
 @each $animal in puma, sea-slug, egret, salamander {
 }
-@each $animal in ((puma),  (sea-slug),  (egret),  (salamander)) {
+@each $animal in ((puma), (sea-slug), (egret), (salamander)) {
 }
 @each $animal in((puma), (sea-slug), (egret), (salamander)) {
 }
-@each $animal in ( (puma),  (sea-slug),  (egret),  (salamander)) {
+@each $animal in ((puma), (sea-slug), (egret), (salamander)) {
```
Comment (evilebottnawi) [inline tests/css_atrule/__snapshots__/jsfmt.spec.js.snap]: @lydell no extra spaces

File: tests/css_atrule/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -2125,7 +2223,7 @@ a {
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 @include mix(1px, 2px, $arg2: 10, 2px 4px 6px);
-@include mix(1px, 2px, $arg2:10, 2px 4px 6px);
+@include mix(1px, 2px, $arg2: 10, 2px 4px 6px);
```
Comment (evilebottnawi) [inline tests/css_atrule/__snapshots__/jsfmt.spec.js.snap]: Always spaces after `:`

**Comment by evilebottnawi** [comment]
Also refactor and more tests

**Pull request merged**